src/fingerprinting/proxy-to-fingerprint-cache.ts

```diff
@@ -0,0 +1,22 @@
+import { Fingerprint } from 'fingerprint-injector';
+import QuickLRU from 'quick-lru';
+
+export class FingerprintToProxyCache {
```

Comment:
this feels like not really needed abstraction? why not use the `QuickLRU` directly?